test/gtest/uct/ib/test_cqe_zipping.cc

```diff
@@ -0,0 +1,157 @@
+/**
+* Copyright (C) NVIDIA Corporation. 2022.  ALL RIGHTS RESERVED.
+*/
+
+#include <common/test.h>
+#include <common/test_helpers.h>
+#include <uct/ib/test_ib.h>
+
+extern "C" {
+#include <uct/api/uct.h>
+#include <uct/ib/base/ib_iface.h>
+}
+
+#define UCT_INSTANTIATE_MLX5_TEST_CASE(_test_case) \
+    _UCT_INSTANTIATE_TEST_CASE(_test_case, rc_mlx5) \
+    _UCT_INSTANTIATE_TEST_CASE(_test_case, dc_mlx5) \
+    _UCT_INSTANTIATE_TEST_CASE(_test_case, ud_mlx5)
+
+class test_cqe_zipping : public test_uct_ib_with_specific_port {
+public:
+    bool is_cqe_zipping_expected()
+    {
+#ifdef ENABLE_STATS
+        return !RUNNING_ON_VALGRIND &&
+               !IBV_PORT_IS_LINK_LAYER_ETHERNET(&m_port_attr);
+#else
+        return false;
+#endif
+    }
+
+    void send_while_possible()
+    {
+        while (am_zcopy() != UCS_ERR_NO_RESOURCE) {
+            ++m_send_cnt;
+        }
+    }
+
+    void wait_for_completion() const
+    {
+        /*
+         * Local deadline prevents hanging inside receive loop in case of
+         * connectivity issues
+         */
+        const ucs_time_t deadline = ucs::get_deadline(60);
+        while ((m_send_cnt != m_recv_cnt) && (ucs_get_time() < deadline)) {
+            progress();
+        }
+        ASSERT_EQ(m_send_cnt, m_recv_cnt);
+    }
+
+    size_t get_total_zipped_count() const
+    {
+        size_t zipped_cqes_count  = 0;
```

Comment:
`size_t zipped_cqes_count = 0;` (remove space)

Reply:
```suggestion
        size_t zipped_cqes_count = 0;
```

Reply:
Sorry for the applying this, but I saw on this code one more time and found that this double space here is for aligning with assignment on the next line, so it shouldn't be removed.